plotter/general_test.go

```diff
@@ -30,6 +30,7 @@ func goldenPath(path string) string {
 // checkPlot checks a generated plot against a previously created reference.
 // If generateTestData = true, it regereates the reference.
 func checkPlot(ExampleFunc func(), t *testing.T, filenames ...string) {
+	rand.Seed(0)
```

Comment:
To make this concurrency-safe, use a rand.Rand instead of the package-level functions.